### ISSUE TITLE
Getting Error at : yarn prod:start

### ISSUE THREAD

**Issue by Shivamnema** [verekia/js-stack-from-scratch#200]
### Chapter:3
I'm try to build and then run the production build using `yarn prod:start` but it gives error. 

 
`error Command failed with exit code 1.`

Please Suggest the changes.

**Comment by verekia** [comment]
Can you give more details? Did you run `yarn prod:build` first? Is there any other information about that error?

**Comment by Shivamnema** [comment]
No. There is no error apart from this one. I think there is a syntax error in this : 
`cross-env NODE_ENV=production pm2 start lib/server && pm2 logs`

Because apart from this all things are working.
